package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "mongoose-delete",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
I think this should be released as a breaking change because it drops support for node < 4 and especially mongodb < 3

Reply:
They still work but it was a hassle getting the travis to pass

Reply:
Did you mean that in reference to my other [comment about travis](https://github.com/dsanel/mongoose-delete/pull/37#discussion_r166305401)?